ironfish/src/utils/error.ts

```diff
@@ -56,11 +56,16 @@ function isNodeError(error: unknown): error is Error & { code: string } {
   return error instanceof Error && 'code' in error && typeof error['code'] === 'string'
 }
 
+function isNotFoundError(error: unknown): error is Error & { code: 'not-found' } {
+  return isNodeError(error) && error.code === 'not-found'
```

Comment:
nit -- should we use `ERROR_CODES.NOT_FOUND` here instead of the string?

Reply:
This leads to that oclif `undefined` issue we saw yesterday.